### PR TITLE
fix(python): add bid expired to the enum

### PR DESCRIPTION
**Pull request by guibescos**

### CODE REVIEW

File: auction-server/src/auction.rs [hunk 1]
```diff
@@ -1583,12 +1583,12 @@ impl ChainStore for ChainStoreEvm {
         tx_hash: Vec<u8>,
     ) -> Result<Option<Vec<<Self::SimulatedBid as SimulatedBidTrait>::StatusType>>> {
         let tx_hash = H256::from_slice(tx_hash.as_slice());
-        let reciept = self
+        let receipt = self
```
Comment (guibescos) [inline auction-server/src/auction.rs:1586]: drive-by typo 😉

File: sdk/python/express_relay/models/base.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ class BidStatus(Enum):
     SUBMITTED = "submitted"
     LOST = "lost"
     WON = "won"
+    EXPIRED = "expired"
```
Comment (danimhr) [inline sdk/python/express_relay/models/base.py:22]: This one is only for svm bids.
